lib/private/RedisFactory.php

```diff
@@ -78,7 +99,24 @@ private function create() {
 				$timeout = 0.0; // unlimited
 			}
 
-			@$this->instance->connect($host, $port, $timeout);
+			if (isset($config['connection_parameters'])) {
+				if (!$isConnectionParametersSupported) {
+					throw new \UnexpectedValueException(\sprintf('Php-redis extension must be equal %s or higher to support connection parameters',
```

Comment:
```suggestion
					throw new \UnexpectedValueException(\sprintf('php-redis extension must be equal %s or higher to support connection parameters',
```
`php-redis` is a proper name